### DISCUSSION TITLE
歌单 | /cn/sing/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1438] [Comments]
https://yihui.org/cn/sing/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
1. 是的，第二可能是王菲。但喜欢听的和能唱的完全是两码事。像《化身为海》我已经听过四百多遍，但还是唱不准。  
2. 我记不住所有的歌词，但我很会联想歌词。  
3. 我自己也早就注意到了这一点。我的解释是周杰伦开始火的时候我刚好在上高中，那时候在埋头学习，没去赶潮流，所以对周杰伦没什么感觉。有些他的歌我大概能哼几句，总体来说不是很熟。  
4. 我听的歌总体偏老年型，太经典的歌可能会是所有人的交集。  
5. 哈哈，我只敢提供点菜服务，点歌真没这个勇气。老实说，我唱歌只是为了自己玩，我喜欢唱，胆子很大，但水平相当菜，零技巧，声带粗糙，声音不稳，音色不好听，不是谦虚。我喜欢假声挑战《左手指月》或者《青藏高原》之类的歌（今天下午刚在朋友家还唱了一遍），但男生的假声一般都比较难听，我的假声类型可能只有唱京剧味儿的《北京一夜》《新贵妃醉酒》还算能驾驭。《左手指月》的最后一句“无二无别”我通常顶不上去，少数高音部分我的假声也会乏力，那几句“啊啊啊”勉强可以踮着脚啊几下，但基本上纯属尖叫、没法听……

> ——原帖发布于 2022-02-13 04:39:53

**Reply by yihui** [reply]
这几个特点还真是一模一样。上次我开长途也是借着噪声掩护唱了一路，不过好多歌词都记不住。本来我是自从十多年前来美国后就没怎么 K 过歌了，过去半年在朋友家又重新变回了麦霸，感觉是可以不吃不喝唱一天的那种（虽然一般只唱一下午），哈哈。

> ——原帖发布于 2022-02-15 18:30:28

[a comment by an automated account is not shown]

**Comment by chuxinyuan** [comment]
感觉都是怀旧金曲，近几年的歌很少，听说怀旧的人都是很重感情的人，哈哈哈……。我在想整成一张表再加上年份字段会不会更方便呢？这样各个阶段的同学一起去 K 歌找起来更方便点。

**Reply by yihui** [reply]
我早就打算自己写 K 歌软件，一直没空。如果是多人一起 K 歌而且麦霸特别多的话，可以每个人预备好自己的歌单，在软件里求个交集。还可以更细致一点，就是每个人在自己的歌单里按喜爱或熟练程度打个分，让软件按歌单交集的平均打分降序排列候选歌曲。
